renderer/src/features/mcp-servers/lib/orchestrate-run-local-server.tsx

```diff
@@ -129,42 +117,49 @@ function transformTypeSpecificData(
  * created secrets from the secret store into a single request object.
  */
 export function prepareCreateWorkloadData(
-  data: FormSchemaRunMcpCommand,
+  data: FormSchemaLocalMcp,
   secrets: SecretsSecretParameter[] = []
 ): V1CreateRequest {
   const request = transformTypeSpecificData(data)
 
-  request.cmd_arguments = data.cmd_arguments || []
-  request.env_vars = mapEnvVars(data.envVars)
-  request.secrets = secrets
+  // Extract and transform fields with proper typing
```

Comment:
nit pick, i guess comments that explain what the code does are not that useful. if there is anything to explain here, it should be maybe _why_ we need to do this

Reply:
yep it makes sense, let me grab this into the next PR, cause I refactor this file in the remote mcp PR